### PR TITLE
adds BluetoothUUID

### PR DESCRIPTION
**Pull request by rachelandrew**
Adding the BluetoothUUID interface in Markdown.

Reviewer @jpmedley

### CODE REVIEW

File: files/en-us/web/api/bluetoothuuid/canonicaluuid/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  - API
+  - Method
+  - Reference
+  - canonicalUUID()
+  - BluetoothUUID
+browser-compat: api.BluetoothUUID.canonicalUUID
+---
+{{APIRef("Bluetooth API")}}
+
+The **`canonicalUUID`**  property of the {{domxref("BluetoothUUID")}} interface returns the 128-bit UUID when passed a 16- or 32-bit UUID alias.
```
Comment (teoli2003) [inline files/en-us/web/api/bluetoothuuid/canonicaluuid/index.md:14]: It is a method, not a property.

**Review by teoli2003 — DISMISSED** [review]

**Comment by teoli2003** [comment]
Before merging this, it would be better to switch the whole Bluetooth API to Markdown. I'm not sure we want to have too much of a mix of md and html files in the same API.

File: files/en-us/web/api/bluetoothuuid/getcharacteristic/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  - API
+  - Method
+  - Reference
+  - getCharacteristic
+  - BluetoothUUID
+browser-compat: api.BluetoothUUID.getCharacteristic
+---
+{{APIRef("Bluetooth API")}}
+
+The **`getCharacteristic()`**  method of the {{domxref("BluetoothUUID")}} interface returns a UUID representing a registered characteristic when passed a name or the 16- or 32-bit UUID alias.
```
[the remark this answers is not in the document]
Comment (jpmedley) [reply files/en-us/web/api/bluetoothuuid/getcharacteristic/index.md:14]: I'm not asking for a lot of detail. I'm thinking in terms of simple handles to help someone new to Bluetooth research on their own.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/bluetoothuuid/canonicaluuid/index.md (hunk 1 above)
Comment (rachelandrew) [reply files/en-us/web/api/bluetoothuuid/canonicaluuid/index.md:14]: > It is a method, not a property.

ah yes I missed this one - the tool made all the methods properties :D

**Review by jpmedley — APPROVED** [review]

**Comment by rachelandrew** [comment]
> Before merging this, it would be better to switch the whole Bluetooth API to Markdown. I'm not sure we want to have too much of a mix of md and html files in the same API.

if we can get the conversion script to work for me, I'll happily do it. At the moment I'd have to do the conversion by hand.

**Comment by rachelandrew** [comment]
@teoli2003 this is blocked on you - now the md conversion is done, I'd like to merge it

**Review by jpmedley — APPROVED** [review]

**Pull request merged**